src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -191,12 +192,15 @@ public static class PublishedContentExtensions
                 throw new ArgumentException(string.Format("The instance parameter does not implement Type '{0}'", type.Name), "instance");
             }
 
+            // Get the accessor for UmbracoContext & ApplicationContext
+            var umbracoApplicationContextAccessor = (IUmbracoApplicationContextAccessor)Activator.CreateInstance(Ditto.GetUmbracoApplicationContextAccessorType());
```

Comment:
The `umbracoApplicationContextAccessor` variable name feels too verbose... let's consider alternatives 😏